docs/advanced-topics/metrics.rst

```diff
@@ -0,0 +1,154 @@
+================
+
+.. note:: Dockstore Workflow Metrics is currently in preview mode. Also note that the screenshots below were taken on our staging site.
+
+What are workflow metrics?
+------------------------------------
+Workflow metrics are metrics of workflow executions on a platform. Metrics include things like execution status and the resources used during the execution. 
+
+Users are able to execute workflows on various platforms using Dockstore's Launch With feature, shown on the right side in the screenshot below.
+
+.. figure:: /assets/images/docs/submit-metrics/workflow-launch-with.png
+    :alt: Launch With Partners
+
+Platforms are able to submit metrics of workflows executed on their platform to Dockstore and we aggregate the metrics and display them in the UI to the users in the :ref:`Metrics tab <How to view metrics>`.
+
+Why would I want to submit workflow metrics?
+--------------------------------------------
+As a platform owner, workflow metrics indicate to others that your platform is compatible with many workflows on Dockstore. Workflow metrics provide valuable information to users, including information about the resources and time needed to run the workflow. It helps the user determine if the workflow is high quality and is very likely to work for others.
+
+.. _How to view metrics:
+
+How do I view workflow metrics?
+-------------------------------
+
+To view workflow metrics for a workflow on Dockstore, you will need to use the ``metrics`` feature flag.
+
+To use the ``metrics`` feature flag, append ``metrics`` to the Dockstore URL as a query parameter. You only need to do this once, unless you refresh/close your browser.
+
+For example, if you're on the https://dockstore.org page, append ``metrics`` such that it looks like this: https://dockstore.org?metrics.
+
+If you're on a page that already contains query parameters, indicated by the presence of a question mark, append ``metrics`` to the URL using an ampersand. For example, if you're on the https://dockstore.org/workflows/github.com/gatk-workflows/seq-format-conversion/BAM-to-Unmapped-BAM:3.0.0?tab=info page, append ``metrics`` such that it looks like https://dockstore.org/workflows/github.com/gatk-workflows/seq-format-conversion/BAM-to-Unmapped-BAM:3.0.0?tab=info&metrics.
+
+After applying the ``metrics`` feature flag, a Metrics tab will appear when viewing a workflow on Dockstore. Click on the Metrics tab and you will see workflow metrics if they are available.
+
+.. figure:: /assets/images/docs/submit-metrics/metrics-tab.png
+    :alt: Metrics tab
+
+How do I submit workflow metrics?
+---------------------------------
+
+.. note:: Submitting workflow metrics is only available for admins, curators, and platform partners. If you're a platform owner and you don't have a platform partner user, please contact us via our `GitHub <https://github.com/dockstore/dockstore/issues>`_ issues or `Gitter <https://gitter.im/ga4gh/dockstore>`_ and we will help you get set up.
```

Comment:
Hmmm, maybe issues or helpdesk on discourse. We don't really monitor gitter consistently